src/harberger-lib/Config.re

```diff
@@ -0,0 +1,58 @@
+[@bs.val] [@bs.scope ("window", "location")]
```

Comment:
Sorry, didn't mean for this change to be in this PR... Let me see if I can fix it